app/src/main/java/com/xa/xpensauditor/Day_wise_visualization.java

```diff
@@ -0,0 +1,204 @@
+package com.xa.xpensauditor;
+
+import static java.lang.System.currentTimeMillis;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.TextView;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'android.widget.TextView' import. Should be before 'androidx.recyclerview.widget.RecyclerView'.